app/models/pending_loan.rb

```diff
@@ -10,4 +10,31 @@ def projects
     end
     output
   end
+
+  def calculate_total_cost
+    total_cost = 0
+    projects.each do |project, loan_amount|
```

Comment:
Unused block argument - `project`. If it's necessary, use `_` or `_project` as an argument name to indicate that it won't be used.